src/python/grpcio/grpc/experimental/aio/_base_channel.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Abstract base classes for Channel objects and Multicallable objects."""
+
+import abc
+from typing import Any, AsyncIterable, Optional
+
+import grpc
+
+from . import _base_call
+from ._typing import DeserializingFunction, MetadataType, SerializingFunction
+
+_IMMUTABLE_EMPTY_TUPLE = tuple()
+
+
+class UnaryUnaryMultiCallable(abc.ABC):
+    """Factory an asynchronous unary-unary RPC stub call from client-side."""
```

Comment:
Suggestion: "Factory for an asynchronous unary RPC stub call on the client-side."

Reply:
Updated to `unary call`/`client streaming`/`server streaming`/`bidirectional streaming` terminology.